IO/NativeSignalHandlers/INativeSignalHandler.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace LocalAdmin.V2.IO.NativeSignalHandlers
+{
+    internal interface INativeSignalHandler
+    {
+        public void Setup();
```

Comment:
This public is kindof useless

Reply:
Perhaps, since the interfaces use the `public` access level by default, but I honestly don’t see the point of removing it, it doesn’t affect the performance, nor the execution logic.

Reply:
But it affects the readability of the code, even github coloring has issues with it

Reply:
Already fixed in https://github.com/northwood-studios/LocalAdmin-V2/pull/4/commits/02a8335375e564ea6d85d51ccfbaa6c837de90ad